app/src/org/commcare/logging/analytics/GoogleAnalyticsUtils.java

```diff
@@ -311,7 +312,8 @@ public static void reportAppManagerAction(String action) {
     }
 
     public static void reportPrivilegeEnabled(String privilegeName, String username) {
-        reportEvent(GoogleAnalyticsFields.CATEGORY_PRIVILEGE_ENABLED, privilegeName, username);
+        reportEvent(GoogleAnalyticsFields.CATEGORY_PRIVILEGE_ENABLED, privilegeName,
+                Base64.encode(username.getBytes()));
```

Comment:
Fuck, sorry, I said this in a confusing way.

We still need to Md5 the value, since otherwise it can still be read. What we want is to MD5 -> Base64, to get a consistent string version of the MD5 hashed value.